kustomize/aks/pub.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: racetrack
+  name: pub
+  labels:
+    app.kubernetes.io/name: pub
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: pub
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: pub
+    spec:
+      securityContext:
+        fsGroup: 200000
+        runAsUser: 100000
+        runAsGroup: 100000
+      hostname: pub
+      subdomain: pub
+      priorityClassName: high-priority
+      containers:
+        - name: pub
+          image: ghcr.io/theracetrack/racetrack/pub:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 7005
+          tty: true
+          env:
+            - name: PUB_PORT
+              value: '7005'
+            - name: LIFECYCLE_URL
+              value: 'http://lifecycle:7002/lifecycle'
+            - name: AUTH_REQUIRED
+              value: 'true'
+            - name: AUTH_DEBUG
+              value: 'true'
+            - name: LIFECYCLE_AUTH_TOKEN
+              value: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzZWVkIjoiY2E3ZWNmODAtZDAxZS00ZjA5LWI4ZjQtMTZiNTMxYjhhOTNhIiwic3ViamVjdCI6InB1YiIsInN1YmplY3RfdHlwZSI6ImludGVybmFsIiwic2NvcGVzIjpbImZ1bGxfYWNjZXNzIl19.Xyg1lwQG4oiA65y-KAkdmCFjnT7BZkfmm66DDGOVZu4'
```

Comment:
Ditto the other token comments.

Reply:
I included steps for generating secrets and tokens. This has been transformed into a placeholder.